srcpkgs/jmeter/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'jmeter'
+pkgname=jmeter
+version=5.1.1
+revision=1
+wrksrc="apache-jmeter-5.1.1"
```

Comment:
Please use `$version` instead of `5.1.1`

Reply:
A java-runtime-environment is missing, so `depends="virtual?java-environment"`, see for example `apache-maven-bin`